internal/docker/container.go

```diff
@@ -369,6 +374,13 @@ func (r *ContainerRunner) collectResults(artifactCfg config.ArtifactDownload, re
 			for _, rep := range r.Reporters {
 				rep.Add(tr)
 			}
+			slackTestResults = append(slackTestResults, slack.TestResult{
```

Comment:
We already have a reporter interface for this exact usecase, so would be great to make use of that one 💪 (see our table & junit reporter in saucectl).

Reply:
Thanks! I used this interface instead of my code.